src/server/pps/cmds/cmds_test.go

```diff
@@ -123,6 +123,62 @@ func TestRawFullPipelineInfo(t *testing.T) {
 		`).Run())
 }
 
+func TestFailedPipelineInfo(t *testing.T) {
```

Comment:
It seems like this test should be geared more towards jobs than pipelines. We are testing a failed job not a failed pipeline. Also, it seems like this is testing the not run / canceled state specifically since that is the only thing being checked, so it probably makes sense to name the test according to that.